task_manager/routes/tasks.py

```diff
@@ -54,31 +54,42 @@ def model_post_create_task():
 
     try:
         db.session.add(new_task)
-        db.session.commit()
```

Comment:
I think this needs to be committed or else the new_task.id doesn't seem to have a value when I tried to do this in a unit test.
Note rollback rolls back the last commit iirc, so I think we will still need this commit.
Can you check?